src/confluent_kafka/src/Producer.c

```diff
@@ -814,6 +891,13 @@ static PyMethodDef Producer_methods[] = {
         { "set_sasl_credentials", (PyCFunction)set_sasl_credentials, METH_VARARGS|METH_KEYWORDS,
            set_sasl_credentials_doc
         },
+        { "topic_new", (PyCFunction)Producer_topic_new, METH_VARARGS|METH_KEYWORDS,
+          "Creates a new topic handle for topic named topic, allows setting a per-topic configuration\n"
```

Comment:
Doesn't this need a function signature in the doc?:
```
          ".. py:function:: topic_new(topic, [conf])\n\n"
```

nit: The standard in the codebase seems to be using `config` rather than `conf`.